### PR TITLE
Added Novelfull Presence

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #4962
Added Novelfull Presence.
n5
n1
n2
n3
n4

### CODE REVIEW

File: websites/N/Novelfull/presence.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  browsingStamp = Math.floor(Date.now() / 1000);
+let search: HTMLInputElement;
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo"
+    },
+    page = window.location.pathname;
+
+  presenceData.startTimestamp = browsingStamp;
```
Comment (Slowlife01) [inline websites/N/Novelfull/presence.ts:13]: move startTimestamp to the init object

File: websites/N/Novelfull/presence.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  search = document.querySelector("#search-input");
```
Comment (Slowlife01) [inline websites/N/Novelfull/presence.ts:14]: why does this variable have to be outside scope if you always reassign it

File: websites/N/Novelfull/presence.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    presenceData.details = "Viewing Novels Genre:";
+    presenceData.state = page.replace("/genre/", "");
+  } else if (page.includes("chapter")) {
+    presenceData.details = document.querySelector(
+      "#container > div.navbar-breadcrumb > div > ol > li:nth-child(2) > h1 > a > span"
+    ).textContent;
+    presenceData.state = document.querySelector(
+      "#chapter > div > div > h2 > a > span"
+    ).textContent;
+    presenceData.startTimestamp;
```
Comment (Slowlife01) [inline websites/N/Novelfull/presence.ts:29]:
```suggestion
```

File: websites/N/Novelfull/presence.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  else if (
+    document.querySelector(
+      "#truyen > div.csstransforms3d > div > div.col-xs-12.col-info-desc > div.title-list > h2"
+    )
+  ) {
+    presenceData.details = "Viewing:";
+    presenceData.state = document.querySelector(
+      "#truyen > div.csstransforms3d > div > div.col-xs-12.col-info-desc > div.col-xs-12.col-sm-8.col-md-8.desc > h3"
+    ).textContent;
+    presenceData.startTimestamp;
```
Comment (Slowlife01) [inline websites/N/Novelfull/presence.ts:46]:
```suggestion
```

**Review by Slowlife01 — DISMISSED** [review]

File: websites/N/Novelfull/presence.ts [hunk 5]
```diff
@@ -0,0 +1,46 @@
+const presence = new Presence({
+    clientId: "914175371744800779"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/N/Novelfull/presence.ts:4]:
```suggestion
  browsingTimestamp = Math.floor(Date.now() / 1000);
```

File: websites/N/Novelfull/presence.ts [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+      "#truyen > div.csstransforms3d > div > div.col-xs-12.col-info-desc > div.title-list > h2"
+    )
+  ) {
+    presenceData.details = "Viewing:";
+    presenceData.state = document.querySelector(
+      "#truyen > div.csstransforms3d > div > div.col-xs-12.col-info-desc > div.col-xs-12.col-sm-8.col-md-8.desc > h3"
+    ).textContent;
+  }
+  if (!presenceData.details) presence.setActivity();
+  else presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/N/Novelfull/presence.ts:45]:
```suggestion
  if (presenceData.details) presence.setActivity(presenceData);
  else presence.setActivity();
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/N/Novelfull/presence.ts [hunk 7]
```diff
@@ -0,0 +1,46 @@
+const presence = new Presence({
+    clientId: "914175371744800779"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingStamp
```
Comment (EncryptedDev) [inline websites/N/Novelfull/presence.ts:9]:
```suggestion
      startTimestamp: browsingTimestamp
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/N/Novelfull/dist/metadata.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+  "category": "other",
+  "logo": "https://i.imgur.com/4UOKOMH.jpg",
+  "thumbnail": "https://i.imgur.com/EgovjIn.jpg",
+  "version": "1.0.0",
+  "color": "#0093c9",
+  "tags": ["novels", "novel", "novefull", "read", "reading"],
+  "url": ["novelfull.com", "www.novelfull.com"],
+  "description": {
+    "en": "Read Books Online Free",
+    "nl": "Lees Online Gratis Boeken"
```
Comment (Bas950) [inline websites/N/Novelfull/dist/metadata.json:17]:
```suggestion
    "nl": "Lees Boeken Gratis Online"
```

**Review by EncryptedDev — DISMISSED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**